apps/angular/src/app/anime-dashboard/anime-dashboard.component.ts

```diff
@@ -0,0 +1,25 @@
+import { Component, inject } from '@angular/core';
+import { CommonModule, DatePipe } from '@angular/common';
+import { MatTableModule } from '@angular/material/table';
+
+import { AnimeService } from '../services/anime.service';
+
+/** Anime dashboard. */
+@Component({
+	selector: 'camp-anime-dashboard',
+	standalone: true,
+	imports: [CommonModule, MatTableModule, DatePipe],
+	templateUrl: './anime-dashboard.component.html',
+	styleUrl: './anime-dashboard.component.css',
+})
+export class AnimeDashboardComponent {
+
+	/** Services. */
```

Comment:
It is a single service
```suggestion
	/** Service. */
```